src/main/resources/application.properties

```diff
@@ -27,6 +27,7 @@ scanner.interval=1000
 # Settings for the registration worker threads
 #----------------
 registration.threads=2
+registration.metadata.filename=metadata.txt
```

Comment:
Why is this not set via an environment variable? Additionally, is it intended that there is no identifying naming scheme(referring to projectID or timestamp) applied here?

Reply:
The working directory is set by an env var WORKING_DIR

Reply:
damn, the review comment just jumped up one line in IntelliJ :::